src/main/java/tests/DataConverterTest.java

```diff
@@ -0,0 +1,61 @@
+package tests;
+
+import core.basesyntax.DataConverter;
+import core.basesyntax.DataConverterImpl;
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.Operation;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class DataConverterTest {
+
+    private final List<String> rawData = Arrays.asList(
+            "type,fruit,quantity",
+            "b,banana,20",
+            "s,banana,100",
+            "p,banana,13",
+            "r,banana,10"
+    );
+    private final DataConverter dataConverter = new DataConverterImpl();
+    private final List<FruitTransaction> transactions = dataConverter.convertToTransaction(rawData);
```

Comment:
The initialization of 'dataConverter' and the call to 'convertToTransaction' should not be in the field declaration section. It's better to initialize and use 'dataConverter' inside each test method to ensure test isolation and to avoid potential side effects from other tests.